packages/nouns-api/src/services/auth.ts

```diff
@@ -64,6 +64,37 @@ class AuthService {
     const allUsers = await prisma.user.findMany();
     return allUsers;
   }
+
+  static async syncUserTokenCounts(to: string, from: string) {
+    try {
+      const toUser = await prisma.user.findUnique({
+        where: {
+          wallet: to,
+        },
+      });
+
+      const fromUser = await prisma.user.findUnique({
+        where: {
+          wallet: from,
+        },
+      });
```

Comment:
There might be a case in which the users just don't yet exist in our database.
We could do an upsert below though to update or create a user record if the transfer is coming or going to a users wallet that doesn't exist in our database